packages/admin/cms-admin/src/contentScope/Controls.tsx

```diff
@@ -24,28 +24,51 @@ export function ContentScopeControls<S extends ContentScopeInterface = ContentSc
     config = {},
 }: ContentScopeControlsProps<S>): JSX.Element {
     const { scope, setScope, values } = useContentScope<S>();
+    const findLabelForScopePart = (scopePart: keyof ContentScopeInterface) => {
+        const label = values.find(({ value }) => value === scope[scopePart])?.label;
+        return label ?? capitalizeString(scope[scopePart]);
+    };
+
+    const label = Object.keys(scope)
+        .map((scopeKey: string) => findLabelForScopePart(scopeKey))
+        .join(" / ");
+
+    const groupedScopes = useMemo(() => {
+        const groupingKey = Object.keys(values[0])[0];
+        const groupedValues: { [key: string]: Array<ContentScopeInterface> } = {};
+        values.forEach((value) => {
+            if (!groupedValues[value[groupingKey].value]) {
+                groupedValues[value[groupingKey].value] = [];
+            }
+            groupedValues[value[groupingKey].value].push(value);
+        });
+
+        return Object.keys(groupedValues).map((group, index) => ({
+            group: { key: groupingKey, value: groupedValues[group][0][groupingKey] },
+            values: groupedValues[group],
+        }));
+    }, [values]);
+
     return (
-        <>
-            {Object.keys(values).map((key) => {
-                const valuesForKey = values[key];
-                const value = valuesForKey.find((c) => c.value === scope[key]);
-                return value ? (
-                    <ContentScopeSelect
-                        key={key}
-                        defaultLabel={config[key]?.label}
-                        icon={config[key]?.icon}
-                        searchable={config[key]?.searchable}
-                        values={valuesForKey}
-                        value={value}
-                        onChange={(newValue) => {
-                            setScope((s) => ({
-                                ...s,
-                                [key]: newValue,
-                            }));
-                        }}
-                    />
-                ) : null;
-            })}
-        </>
+        <ContentScopeSelect
+            label={label}
+            icon={config.domain?.icon}
+            searchable={config.domain?.searchable}
```

Comment:
You can't hardcode domain here. Not every project will have a scope part "domain". Also, the config should be refactored to the new approach